include/nbl/system/ICancelableAsyncQueueDispatcher.h

```diff
@@ -0,0 +1,179 @@
+#ifndef __NBL_I_CANCELABLE_ASYNC_QUEUE_DISPATCHER_H_INCLUDED__
+#define __NBL_I_CANCELABLE_ASYNC_QUEUE_DISPATCHER_H_INCLUDED__
```

Comment:
sorry to be a prick, but something about cancelable with one `l` looks off
https://grammarist.com/spelling/cancel/